### PR TITLE
[webgpu] Dump @webgpu/types to 0.1.15

### PR DESCRIPTION
**Pull request by haoyunfeix**
And replace dispatch() with dispatchWorkgroups()

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6378)

### CODE REVIEW

**Comment by haoyunfeix** [comment]
@qjia7 @xhcao @axinging @gyagp PTAL

File: tfjs-backend-wasm/tsconfig.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "node_modules/"
   ],
   "compilerOptions": {
+    "typeRoots": [ "../node_modules/@webgpu", "./node_modules/@types"],
```
Comment (gyagp) [inline tfjs-backend-wasm/tsconfig.json:10]: Why is wasm backend dependent on WebGPU types?

**Comment by gyagp** [comment]
Need to change Dump to Bump in commit subject.

**Comment by haoyunfeix** [comment]
Open new PR at https://github.com/tensorflow/tfjs/pull/6388

**Pull request closed without merging**